js/model.js

```diff
@@ -7,21 +7,165 @@ const LETTER_FREQUENCY = [
   83.559, 92.915, 95.673, 96.651, 99.211, 99.361, 101.355, 101.432
 ];
 
-/** @record */
+/** @const {!Array<String>} */
+const WORDS = ['patrick'];
+
+class Point {
+  /**
+   * @param {number} x
+   * @param {number} y
+   */
+  constructor(x, y) {
+    /** @const {number} */
+    this.x = x;
+    /** @const {number} */
+    this.y = y;
+  }
+
+  /**
+   * Returns the difference between two points
+   * @param {!Point} pointB
+   * @returns {!Point}
```

Comment:
nitpick: Google JS typically uses `@return` instead of `@returns`.

They are functionally equivalent to Closure Compiler. But I think it's useful to get in the habit of writing `@return` instead of `@returns` for consistency, especially when you make Closure Compiler PRs.

Reply:
Got it. Updated.